docs/feature-flags.md

```diff
@@ -57,3 +57,16 @@ At some point, a feature flag will be turned on permanently. To do this, add a `
 ```
 
 Before publishing the next release, all remnants of permanently enabled feature flags shall be removed from the code base.
+
+
+## Turning on/off feature flags with the Experiments tab
+
+Feature flags can be managed via a hidden "Experiements" tab within the Stories WordPress nav. There you can turn on/off all experiments defined in `includes/Experiments.php`.
+
+To turn it on, edit `web-stories.php` to switch `WEBSTORIES_DEV_MODE` from `true`. This will enable the Experiments tab, but make sure to set this back to `false` after you're done in order to not inadvertently commit this change.
```

Comment:
The recommendation is to actually add `define( 'WEBSTORIES_DEV_MODE', true );` to your `wp-config.php` file because of that.

Reply:
Ah ha, perfect thank you! That is a much better solution. I'll make the change.

Reply:
So this makes more sense to me, but how would I go about doing this with our automated docker setup? Is this something we could implement as part of the local env setup?

Reply:
In the docker setup this should be on by default already. Is this not the case for you?